### ISSUE TITLE
Deleting tmp files

### ISSUE THREAD

**Issue by mer23** [jooby-project/jooby#893]
Uploads to server are being saved to /tmp, but are never deleted. Is there a way to configure jooby to do so after it finished working with a file?

**Comment by jknack** [comment]
They supposed to be deleted. Can you please post an usage example?

**Comment by mer23** [comment]
My bad. Reading the docs I realized that I wasn't calling `close()` on my `Upload`s.

Now I'm looking for a more generic way of closing Uploads once I'm done processing my requests, so I don't need to close each one of them individually, something like:
```
complete("*", (req, rsp, cause) -> {
           req.getAllfiles().forEach(f -> f.close());
});
```
but I don't see a way of getting all uploads from req without knowing their names. Any ideas?
TIA

**Comment by jknack** [comment]
Correct, you must call `close()` method.

There is no way of doing this today, but also:

1. there is a mem policy where the file isn't always written to disk
2. on large file uploads you delay the close (free resource) until the request is done, you are introducing a (temporary) leak.
3. The consequences of calling file/files multiple times is unknown. It might give you the same reference or not.

In short: read your file once and make sure to close it (sorry)
